### PR TITLE
fix wrong letter and background color of the channel/group avatar

### LINKED ISSUE #19944: Wrong letter and background color of the channel/group avatar
Wrong letter and background color of the channel avatar
https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=16876-89050&mode=design&t=XlIs4xoeCOqHz6b3-4

image

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/19944

### Designs:

Community Avatar: https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=16876-89050&mode=design&t=kXzE8JgtXAcU4AIp-0

Group Avatar: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=179-9760&mode=design&t=pV0pn0UiszwVIz2z-0

### Testing

Please check community and group avatar in chat screen both themes.

status: ready

### CODE REVIEW

File: src/quo/components/avatars/group_avatar/view.cljs [hunk 1]
```diff
@@ -50,14 +49,6 @@
             {:size  :paragraph-1
              :style (dissoc (style/avatar-identifier theme) :font-size)}
             emoji])
-         chat-name
```
Comment (Parveshdhull) [inline src/quo/components/avatars/group_avatar/view.cljs:53]: group avatar don't support initials
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=179-9760&mode=design&t=pV0pn0UiszwVIz2z-0

Comment (ibrkhalil) [reply src/quo/components/avatars/group_avatar/view.cljs:53]: I think the same component was used for community channels too, That's why it had initials.

**Review by ibrkhalil — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

**Review by clauxx — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
@Parveshdhull, thanks for the PR. No problem on my part.


Hi @Francesca-G, could you take a look at the current PR please?

**Review by Francesca-G — APPROVED** [review]
Avatar looks good to me 👍

**Pull request merged**